torch/ao/quantization/fx/_model_report/detector.py

```diff
@@ -0,0 +1,406 @@
+from typing import Any, Dict, Set, Tuple
+
+import torch
+import torch.nn as nn
+from abc import ABCMeta, abstractmethod
+from torch.ao.quantization.fake_quantize import FakeQuantize
+from torch.ao.quantization.fx.graph_module import GraphModule
+from torch.ao.quantization.observer import ObserverBase
+from torch.ao.quantization.qconfig import QConfig
+from torch.nn.qat.modules.conv import _ConvNd as QatConvNd
```

Comment:
maybe write down concrete modules instead of _ConvNd, e.g. nn.qat.Conv1d, nn.qat.Conv2d, nn.qat.Conv3d